openbas-framework/src/main/java/io/openbas/model/Expectation.java

```diff
@@ -4,10 +4,15 @@
 
 public interface Expectation {
 
-    EXPECTATION_TYPE type();
-    Double getScore();
-    default boolean isExpectationGroup() {
-        return false;
-    }
-    String getName();
+  EXPECTATION_TYPE type();
+
+  Double getScore();
+
+  default boolean isExpectationGroup() {
+    return false;
+  }
+
+  String getName();
+
+  Long getExpirationTime();
```

Comment:
/**
 * Gets the expiration time.
 *
 * @return the expiration time in minutes ? seconds ? (or milliseconds)
 */
Long getExpirationTime();